### ISSUE TITLE
What's happen with github-actions[bot] <notifications@github.com> 

### ISSUE THREAD

**Issue by knupel** [benfry/processing4#204]
Hi people,

since yesterday I've received for of thousand mail from ` github-actions[bot]` from Processing it'a a big spam... and now each hour a received hundred mails. Any people are on the same situation? For now it's complicated for me to read the good email from my mail box. For sur I can stop information from Processing but I don't want :)  Is it a problem from github or from Processing ?

Et voilà

**Comment by benfry** [comment]
In order to clean up this ancient 3.x repository, I've set issues that have been resolved to lock after 30 days. I've been having trouble with a high number of comments on issues that are 4 and 5 years old, with folks reporting new bugs that kinda sorta sound similar but are unrelated. (And in particular, if the issue is *closed*, then it's been resolved.)

I'd been doing this manually as they came up (locking issues that have long since been resolved), but it became too much for me to manage, so I turned on the bot instead. 

But yikes—were you subscribed to email notifications for *all* events on the Processing project? If so, you'll get quite a bit of mail. If not everything, you may be receiving several emails for issues that have been closed that you participated in. Which in your case, may be quite a few. :)

Bottom line: 1) I wouldn't recommend being signed up for email notifications for every bit of activity in here, and 2) it should sort itself out soon, but in the meantime you can also just turn off email notifications for 24-48 hours while that's happening.

**Comment by knupel** [comment]
Thx again for yours lights boss, I go to close few notifications !!!! I understand now, sometimes even humans need a bot to help them !!!

**Comment by benfry** [comment]
Thanks!
